design/README.md

```diff
@@ -1,48 +1,75 @@
 # System Architecture
 
--------------
-
-This document contains description of ventilator design implementation and closure of issues which
+This section of the repository contains the description of ventilator design implementation and closure of issues which
 apply at the system level, such as architecture configuration.
 
-## Overview
+## Design Motivations
 
-A ventilator is a device which supports the breathing of a patient who cannot generate
```

Comment:
Rephrase to be concise:

Ventilators are devices that support patient breathing. The device must generate sufficient pressure to overcome lung and machine resistance. Pressure must vary over time as the lungs fill and then deflate. To improve oxygenation, ventilators also almost universally apply some back-pressure to the lungs (PEEP).

There are dozens of ways to design a ventilator and each has trade-offs. This document describes the design decisions and reasoning that led to the final design implementation.

The first design decision to address is how the ventilator should generate the pressure required to drive air into the lungs.

Displacement type devices provide a mechanical compression of a fixed volume to drive air into the lungs. These are commonly employed in existing ventilators, and they provide a high degree of volumetric accuracy and precise control of flow. There are two challenges here. The first is a question of supply chain. The volume of air which must be delivered into a lung is relatively large, as far as traditional cylinders go. The lubricant and gasket material must be oxygen compatible, and relatively expensive and sensitive drive mechanisms are required. The designs end up rather heavy and expensive. Bellows-type devices can be helpful in addressing the volume issue, but are very challenging to design for the millions of cycles required, in addition to being difficult to source.